dev-packages/rollup-utils/bundleHelpers.mjs

```diff
@@ -51,6 +50,7 @@ export function makeBaseBundleConfig(options) {
       intro: () => {
         return 'exports = window.Sentry || {};';
       },
+      banner: 'window.SENTRY_SDK_SOURCE = window.SENTRY_SDK_SOURCE || "cdn";',
```

Comment:
So I changed this to completely remove the build-time const replacement, and instead inject this for the main CDN bundles. So the logic is now: Take `window.SENTRY_SDK_SOURCE || 'npm'`, leading to npm bundle size not increasing.

The previous approach favored bundle size for the CDN bundles, but IMHO this is the "better" approach as it keeps CDN-specific stuff in the CDN bundles 🤔

Reply:
The CDN bundles now look like this at the top:

```js
/*! @sentry/browser 8.26.0 (8205969a4) | https://github.com/getsentry/sentry-javascript */
window.SENTRY_SDK_SOURCE = window.SENTRY_SDK_SOURCE || "cdn";
var Sentry = (function (exports) {
 // remaining code...
```